packages/web/src/pages/edit-page/EditTrackPage.tsx

```diff
@@ -97,11 +85,7 @@ export const EditTrackPage = (props: EditPageProps) => {
         }
       })
     } else {
-      updateTrack({
-        trackId,
-        userId: currentUserId!,
-        metadata
-      })
+      dispatch(editTrack(trackId, metadata))
```

Comment:
would dispatching this action inside the query mutation work?

Reply:
it's actually just inside a normal function, we are replacing the mutation with sagas here because edit-track is mutation is not ready for prime-time yet

Reply:
ooh yea i meant why not use the mutation and call sagas like we're doing for other migrations but i see it's just not ready yet